distribution/dashboard/ui/src/pages/Tenant/Detail/Topo/index.tsx

```diff
@@ -1,4 +1,29 @@
+import { getNSName } from '@/pages/Cluster/Detail/Overview/helper';
+import TopoComponent from '@/pages/Cluster/Detail/Topo';
```

Comment:
The tenant topology depends on the cluster topology. This seems inappropriate. If they all depend on it, it needs to be placed in the common part.

Reply:
I am making an effort.